backend/routes/canvas.go

```diff
@@ -19,7 +20,16 @@ func initCanvas(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if core.ArtPeaceBackend.Databases.Redis.Exists(context.Background(), "canvas").Val() == 0 {
+	// Get round number from query params
+	roundNumber := r.URL.Query().Get("round")
```

Comment:
It might be better to add the round number to the [canvas configs](https://github.com/keep-starknet-strange/art-peace/tree/main/configs) then use that here as `core.ArtPeaceBackend.CanvasConfig.Round`.

Reply:
You can set the value in the config to round 2 for now